src/festim/exports/surface_temperature.py

```diff
@@ -0,0 +1,51 @@
+import csv
+from dolfinx import fem
+import ufl
+from .surface_quantity import SurfaceQuantity
+
+
+class AverageSurfaceTemperature(SurfaceQuantity):
+    """Exports the average temperature on a given surface.
+
+    Args:
+        surface (int or festim.SurfaceSubdomain): the surface subdomain
+        filename (str, optional): name of the file to which the average surface temperature is exported
+
+    Attributes:
+        temperature_field (fem.Constant or fem.Function): the temperature field
+        surface (int or festim.SurfaceSubdomain): the surface subdomain
+        filename (str): name of the file to which the surface temperature is exported
+        t (list): list of time values
+        data (list): list of average temperature values on the surface
+    """
+
+    def __init__(self, surface, filename: str = None) -> None:
+        self.surface = surface
+        self.filename = filename
+
+        self.temperature_field = None
+        self.t = []
+        self.data = []
+        self._first_time_export = True
```

Comment:
I would rather have use the super class init for consistency

```suggestion
    def __init__(self, surface, filename: str = None) -> None:

        self.temperature_field = None
        super().__init__(field=None, surface=surface, filename=filename)
```